gradle.properties

```diff
@@ -1,6 +1,6 @@
 # Gradle
+org.gradle.jvmargs=-Xmx1536m
 org.gradle.caching=true
-org.gradle.configureondemand=true
```

Comment:
Does configure on demand not work or doesnt apply since this project only has 1 module?

Reply:
It doesn't apply since this project only has 1 module.